pcsx2/gui/GlobalCommands.cpp

```diff
@@ -184,15 +184,15 @@ namespace Implementations
 			default: break;
 		}
 
-		Console.WriteLn(L"(GSwindow) Aspect ratio: %s.", WX_STR(arts));
+		OSDlog( Color_StrongBlue, true, "(GSwindow) Aspect ratio: %s.", WX_STR(wxString(arts)));
```

Comment:
Doesn't work on Windows - The Console Logger has some annoying quirks (which do need to be fixed at some point), so you either need to use `.c_str()` in OSDlog (as I previously suggested) or do my no pragma suggestion, which should also take care of it.